### PR TITLE
Adjust the character's gaze so that it does not peek into blocks above

### LINKED ISSUE #2138: The player's head can bump slightly too far into the ceiling, seeing through the world
Only noticed this while testing another PR but am thinking it is a consequence of #2133 since that had to do with the player's vision, or the location of as much :-)

Easy way to reproduce: blast a horizontal tunnel with the railgun and walk into it, jumping into the low hanging ceiling repeatedly. It looks like the players eyes are too far up in the character's physics capsule or what not?

@Josharias this one is all yours, probably an easy fix to adjust the height/offset or something, maybe with your next round of related changes, no rush :-)

Edit: Was not present in Alpha RC2, so I figure it is super recent.

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/30033831-the-player-s-head-can-bump-slightly-too-far-into-the-ceiling-seeing-through-the-world?utm_campaign=plugin&utm_content=tracker%2F5279&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F5279&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by Josharias**
Fixes #2138

### CODE REVIEW

**Comment by GooeyHub** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.terasology.org/job/TerasologyPRs/406/
Hooray Jenkins reported success with all tests good!

**Comment by msteiger** [comment]
Is there a one-line-explanation on why this does the trick?

**Comment by Josharias** [comment]
Thank you for calling this out.

The bug #2138 was caused by a taller eyeball height introduced with the camera entity.

This PR increases the character physics capsule to 1.8 so that there is some padding around the eyeball height when bonking into the ceiling.  Also, decrease the eyeball height back to 1.6... I think (the character location seems to be 0.9 above ground. So 0.9+0.7=1.6).

File: engine/src/main/resources/assets/prefabs/player/player.prefab [hunk 1]
```diff
@@ -21,7 +21,7 @@
         "maxGroundSpeed" : 3,
         "distanceBetweenFootsteps" : 1.5,
         "distanceBetweenSwimStrokes" : 2.5,
-        "height" : 1.6,
+        "height": 1.8,
```
Comment (Josharias) [inline engine/src/main/resources/assets/prefabs/player/player.prefab]: This is what the physics character capsule uses for its size.

File: engine/src/main/resources/assets/prefabs/player/player.prefab [hunk 2]
```diff
@@ -31,7 +31,11 @@
         "hideFromOwner" : true
     },
     "BoxShape" : {
-        "extents" : [1.5, 1.6, 1.5]
+        "extents": [
+            1.5,
+            1.8,
```
Comment (Josharias) [inline engine/src/main/resources/assets/prefabs/player/player.prefab]: Changed this for consistency with the other height.

**Pull request merged**